Installation/With-Python-only.md

```diff
@@ -29,6 +29,18 @@ python3 -m pip install -Ur requirements.txt
 python3 start.py tentacles --install --all
 ```
 
+
+{% hint style="note" %}
+On some setup like 32-bit ARM architectures, you might get a `rust` related error while running `python3 -m pip install -Ur requirements.txt` when installing `cryptography`.
+If this happens, you need to install the `rust compiler`: `cryptography` is coded in `rust`.
```

Comment:
Or to use the environment variable to prevent rust build

Reply:
It was a temporary variable only for a specific version of cryptography if i remember well

Reply:
I don't think so

Reply:
https://github.com/pyca/cryptography/commit/82a12a54b602c994cd0361d1b95bcb4047d67a31#diff-55df9d2f8b8d40391d9d451a1e8155292465d48c52e78dd34655be834cd3dc9fL135

Reply:
Oh right 👍